src/OpenTelemetry.Exporter.Prometheus.Shared/PrometheusSerializerExt.cs

```diff
@@ -16,7 +16,13 @@
 
 using OpenTelemetry.Metrics;
 
+#if PROMETHEUS_ASPNETCORE
+namespace OpenTelemetry.Exporter.Prometheus.AspNetCore
```

Comment:
Curious - why do we need to have different namespaces (for something internal)? I feel this is breaking the abstraction (e.g. the PrometheusSerializer should have no knowledge about the exproter/listener).

Reply:
removed